week7-dom/index.html

```diff
@@ -0,0 +1,193 @@
+---
+layout: default
+permalink: /week7-dom/
+---
+<h2>Quiz 7 DOM: events and callback function</h2>
+
+<h4>Please copy and paste the code to the <a href="https://alpha.editor.p5js.org/">p5 web editor</a> to do the following questions.</h4>
+<p>
+  <iframe src="https://alpha.editor.p5js.org/embed/rym0wT6R" width=200 height=200></iframe>
+  <br/>
+
+  1. Write a program that implements the above sketch. First, create a <code>p</code> element. Then use <code>.positon()</code> function to update the element's x, y positon. Then use <code>.html</code> function to update its content. In the end, use <code>.style</code> function to change its background color and add paddings to the element.
+</p>
+
+<script type="text/p5" data-preview-width="400">
+// Please copy and paste the code to the p5 web editor to do the following questions.
+var p;
+var x = 50;
+var y = 50;
+
+function setup() {
+  // create a p element.
+  p = __________('p', 'Waiting.');  
```

Comment:
this should be using `createP()` or `createDiv()` over `createElement('p',____)`